src/com/dotcms/plugin/saml/v3/SamlUtils.java

```diff
@@ -722,5 +726,82 @@ public static Element toElement (final XMLObject object)  {
         }
     } // invokeMessageHandlerChain.
 
+    /**
+     * Check if we can read the KeyStore from file using the SAML properties.
+     *
+     * @param samlProperties {@link Properties} with the values needed.
+     * @return
+     */
+    public static Set<String> validateKeyStore(Properties samlProperties) {
+        Set<String> otherErrors = new HashSet<>();
+        if ( samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_PATH) != null
+            && samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_PASSWORD) != null
+            && samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_TYPE, KeyStore.getDefaultType()) != null) {
+
+            final String pathToKeyStore = samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_PATH);
+            final String keyStorePassword = samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_PASSWORD);
+            final String keyStoreType = samlProperties.getProperty(DotSamlConstants.DOTCMS_SAML_KEY_STORE_TYPE, KeyStore.getDefaultType());
+
+            try {
+                SamlUtils.readKeyStoreFromFile(pathToKeyStore, keyStorePassword, keyStoreType);
+            } catch (DotSamlException e){
+                otherErrors.add("Error reading Key Store");
+            }
+        }
+        return otherErrors;
```

Comment:
variables can be reused here, something like this: https://gist.github.com/dsilvam/67124d8af6d843766e22c18394e76df3